event/handler.go

```diff
@@ -127,6 +135,7 @@ func (h *ImagePublishedHandler) Handle(ctx context.Context, event *ImagePublishe
 	imageDownload, err = h.ImageAPICli.PutDownloadVariant(ctx, "", h.AuthToken, "", event.ImageID, event.ImageVariant, imageDownload)
 	if err != nil {
 		log.Event(ctx, "error putting image variant to API", log.ERROR, log.Error(err), logData)
+		h.setImageStatusToFailed(ctx, event.ImageID, "error putting updated image variant to API")
```

Comment:
Extra context needed on which variant couldn't be updated
```suggestion
		h.setImageStatusToFailed(ctx, event.ImageID, fmt.Sprintf("error putting updated image variant '%s' to API", event.ImageVariant))
```